scripts/build-circt.sh

```diff
@@ -11,8 +11,13 @@ CIRCT_BUILD=${JLM_ROOT_DIR}/build-circt
 CIRCT_INSTALL=${JLM_ROOT_DIR}/usr
 LLVM_LIT_PATH=`command -v lit || true`
 
-LLVM_VERSION=18
-LLVM_CONFIG_BIN=llvm-config-${LLVM_VERSION}
+# Include global shell configuration
+if [ -f ${JLM_ROOT_DIR}/shell.config ]; then
+	source ${JLM_ROOT_DIR}/shell.config
+else
+	echo "Please use configure.sh to configure jlm."
+	exit
+fi
```

Comment:
Once you have the shell.config file, you can also simplify the if-then-else to a simple source.